### PR TITLE
Admin - pagination changes

### PR DESCRIPTION
**Pull request by puneetmoonraft**
- Admin - Dropdown to select user safe, application safe and shared safe to be removed
- To add tabs for navigating between User safe, Shared Safe and Application Safe.
- To hide the option for user to shift between tabular view and tile view. By default only tile view will be visible.
- To display Show more options below which will be absolutely positioned:

### CODE REVIEW

File: tvaultui/src/app/Features/admin/admin.scss [hunk 1]
```diff
@@ -27,10 +27,10 @@
     height: 100% !important;
 }
 .admin.slim-container {
-    width: 100%;
+    // width: 100%;
```
Comment (sivanag1974) [inline tvaultui/src/app/Features/admin/admin.scss]: Commented lines. Are they required?

File: tvaultui/src/app/Features/admin/admin.scss [hunk 2]
```diff
@@ -27,10 +27,10 @@
...
     height: auto;
-    margin: auto;
-    padding: $gutterspace-top-bottom;
+    // margin: auto;
```
Comment (sivanag1974) [inline tvaultui/src/app/Features/admin/admin.scss]: Commented lines

File: tvaultui/src/app/Features/admin/admin.controller.js [hunk 3]
```diff
@@ -38,9 +38,9 @@
 
         $scope.tableOptions = [
             {
-                "type": "All safes",
-                "value": ""
-            }, {
+            //     "type": "All safes",
```
Comment (sivanag1974) [inline tvaultui/src/app/Features/admin/admin.controller.js]: Commented line

File: tvaultui/src/app/Features/admin/admin.scss [hunk 4]
```diff
@@ -27,10 +27,10 @@
...
+    // padding: $gutterspace-top-bottom;
```
Comment (sivanag1974) [inline tvaultui/src/app/Features/admin/admin.scss]: Commented line

File: tvaultui/src/app/Features/admin/admin.scss [hunk 5]
```diff
@@ -116,6 +116,27 @@
...
 
+.flex-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.admin-tabs{
+    // width:60vw;
```
Comment (sivanag1974) [inline tvaultui/src/app/Features/admin/admin.scss]: Is it required?

File: tvaultui/src/app/Features/admin/admin.scss (hunk 1 above)
Comment (puneetmoonraft) [reply tvaultui/src/app/Features/admin/admin.scss]: Not required. I updated the PR.

**Pull request merged**